AuthorApp/Tools/IsNewToTextConverter.cs

```diff
@@ -0,0 +1,49 @@
+﻿using AuthorApp.Model;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Data;
+
+namespace AuthorApp.Tools
+{
+    public class IsNewToTextConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            if (value is Author)
```

Comment:
We can make it simpler:
if (value is Author author)
{
   if (author.IsNew)
      return "New author"